src/Config.js

```diff
@@ -2,19 +2,21 @@
 // configured.
 // mount is the URL for the root of the API; includes http, domain, etc.
 
+import { isBoolean, isString } from 'lodash';
+import net from 'net';
 import AppCache from './cache';
 import DatabaseController from './Controllers/DatabaseController';
-import net from 'net';
+import { logLevels as ctrlLogLevels } from './Controllers/LoggerController';
```

Comment:
To simplify, can you just import controller and reference it directly below where you need its logLevels property?
 
```suggestion
import LoggerController from './Controllers/LoggerController';
```

Reply:
Sorry, but I'm not sure about how you want me to do that : ```logLevels``` is a not property of LoggerController but a constant...

Reply:
Got it, I'd just give it a more descriptive name then, like `allowedLogLevels`

Reply:
Got it, then I would just give it a more descriptive name like `validLogLevels`